src/libs/utils.ts

```diff
@@ -35,18 +37,12 @@ export const makePrettyDate = (dateString: number | string | Date): string => {
   return cond(
     [isToday(date), () => 'Today'],
     [isYesterday(date), () => 'Yesterday'],
-    [differenceInCalendarMonths(date, Date.now()) <= 6, () => dateFormat.format(date)],
+    [differenceInCalendarMonths(date, Date.now()) <= 6, () => formatDate(date)],
     () => monthYearFormat.format(date)
   );
 };
```

Comment:
Didn't move the `makePrettyDate` function. It's only used in a few places and I wanted to ask UX about it. It doesn't make sense to me. It returns a relative date sometimes and an absolute others. It seems like it would be better to use one or the other.